### PR TITLE
Update for has_visit_observation and is_real_time

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: changelogs/current-change_log.md [hunk 1]
```diff
@@ -21,9 +21,11 @@ Linker table for the many-to-many relationship between problem lists and hospita
...
 Table           | Attributes added | Attributes removed | 
 :-- |:-- |:--
-patient_condition     | added_date |
-patient_condition     | resolution_date |
-patient_condition     | is_deleted |
+patient_condition      | added_date |
+patient_condition      | resolution_date |
+patient_condition      | is_deleted |
+visit_observation_type | has_visit_observation |
+visit_observation_type | is_real_time |
```
Comment (stefpiatek) [inline changelogs/current-change_log.md:28]: I think as well adding a summary or explanation of why the tables have changed would be useful.

**Review by stefpiatek — APPROVED** [review]

File: changelogs/current-change_log.md (hunk 1 above)
Comment (skeating) [reply changelogs/current-change_log.md:28]: agreed

**Review by skeating — APPROVED** [review]

File: changelogs/current-change_log.md (hunk 1 above)
Comment (t-young31) [reply changelogs/current-change_log.md:28]: done!
please feel free to push any clarifying changes. documentation is not my forté

File: changelogs/current-change_log.md [hunk 2]
```diff
@@ -21,17 +21,24 @@ Linker table for the many-to-many relationship between problem lists and hospita
...
 
 
 ### Changes/fixes
 
+- Adds columns to patient_condition to accommodate problem lists 
+- Adds a `has_visit_observation` column to `visit_observation_type` as a flag for whether this type has underlying data
+- Adds a `is_real_time` column to `visit_observation_type` as a flag whether the data is being updated in real time
```
Comment (stefpiatek) [inline changelogs/current-change_log.md:35]:
```suggestion
- You can now know if a visit observation has any data (`has_visit_observation` column) and if it is being updated in real time (`is_real_time` column)
```

**Pull request merged**